lib/matplotlib/colors.py

```diff
@@ -529,7 +529,18 @@ def __call__(self, X, alpha=None, bytes=False):
         if vtype == 'scalar':
             rgba = tuple(rgba[0, :])
         return rgba
-
+    
+    def __copy__(self):
+        """Create new object with the same class and update attributes
+	If object is initialized, copy the elements of _lut list
+	"""
+	cls = self.__class__
```

Comment:
The low layer of python is presumably faster than the normal python code. Another way of doing this would be creating a normal Colormap object and manually copying each attribute.